Added buy and sell functionality

nenadV91 opened the pull request:
# Summary

Part of the #574

This PR adds buy and sell buttons to the tokens table
Also adds some smaller style updates

Desktop
![Screenshot from 2022-06-07 20-12-53](https://user-images.githubusercontent.com/34926005/172453535-4c1a2ee3-c189-4a64-8364-c7ef565fe765.png)

Mobile
![Screenshot from 2022-06-07 20-13-17](https://user-images.githubusercontent.com/34926005/172453589-ad10ddfd-eeb0-4fd6-a895-bfeca54a690f.png)

elena-zh commented:
Hey @nenadV91 , great changes!
However, some issues from my side: 
1. Tokens have become not clickable, so it is impossible to open a single token page
2. When scroll pages in a mobile view, it would be nice to return the list to show token names. See the [video](https://watch.screencastify.com/v/N9P1a9PPisTSjkPPMGNY): I see Buy and Sell buttons only when I flip pages 
3. I have got the app crash when I pressed on Sell USDT token in Rinkeby (reproduced only once)
![sell USDT](https://user-images.githubusercontent.com/70885163/172803619-f8aee090-d223-4d41-9cc8-2229768a4a3b.jpg)
4. Balance sorting arrow is shifted in a mobile view now
![image](https://user-images.githubusercontent.com/70885163/172804523-70eeaf50-869d-4932-ad88-089274caa7b3.png)

Thanks!

nenadV91 commented:
@elena-zh Thanks for the review
1. I removed this on purpose for now since we don't plan to do this right now, maybe in the future
2. By default you should be able to see the token names and then there is a horizontal scrollbar to move table content left and right https://watch.screencastify.com/v/nrjDu1BCZgFjwXUjbWO4
3. I wasn't able to reproduce this. Based on the error it could be that when you tried this at that point the PR was building but I'm not 100% sure. I suggest we test this later on few times but I think it should be fine
4. Fixed :+1:

elena-zh commented:
Hey @nenadV91 , thank you for the explanation!

3 I understand the way of the table implementation in a mobile view, but my issue was a bit for another case: I'd like to have a functionality that returns a scroll to the default state when a user flips pages. 
See the video: user sees only Buy and Sell buttons, not token names, when he/she flips pages: https://watch.screencastify.com/v/ZIjpPx4ANrrqUpm6ZUV8

4 Verified and LGTM!


Thanks!

nenadV91 commented:
I'm going to merge this and we can do a review of code afterwards.
